components/02-elements/button/button.config.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
     {
       name: 'secondary',
       context: {
-        class: 'button--secondary'
+        class: 'button--secondary',
+        text: 'I am button secondary'
       }
     },
     {
```

Comment:
please add attribute `aria-label` to button--icon variant

Reply:
Done